pkg/pipeline/encode/encode_json_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2022 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package encode
+
+import (
+	"encoding/json"
+	"github.com/netobserv/flowlogs2metrics/pkg/config"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func initNewEncodeJson(t *testing.T) Encoder {
+	newEncode, err := NewEncodeJson()
+	require.Equal(t, err, nil)
```

Comment:
I think the following is more expressive:
```go
require.NoError(t, err)
```

Reply:
done